doc/api/process.markdown

```diff
@@ -873,6 +883,38 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.send(message[, sendHandle])
+
+* `message` {Object}
+* `sendHandle` {Handle object}
+
+When node is spawned with an IPC channel attached, it can send messages to its
+parent process using `process.send()`. Each will be received as a
+['message'](child_process.html#child_process_event_message)
```

Comment:
Why? There are over a hundred examples in the API docs of the markup above, including one to this specific event.

Separating out to a ref at the bottom is useful when there are multiple uses made of that ref, otherwise I don't see the point.

If there is a stylistic or technical reason to change all markdown refs to the format you used in #2142, writing a tool to convert them all would be a good way, but that is out of scope here.